nexus-repository-apk/src/main/java/org/sonatype/nexus/plugins/apk/internal/ApkSecurityFacet.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Sonatype Nexus (TM) Open Source Version
- * Copyright (c) 2018-present Sonatype, Inc.
+ * Copyright (c) 2019-present Sonatype, Inc.
```

Comment:
was it changed automatically?

Reply:
@aornatovskyy No this was changed manually, most of the documentation was carried over from other community formats and repurposed for the APK community format. I put 2019 since this is a brand new community format (i based the date on first commit year)

Reply:
No legalese here but I feel like that's appropriate fwiw, it should be creation and beyond.

Reply:
This is correct, it should be the year the plugin/project was created.